test/files/e2e-test-pod.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: "Template"
+metadata:
+  name: elasticsearch-operator-test-pod-template
+  annotations:
+    description: "Template for creating pod for elasticsearch operator test image."
+    tags: "infrastructure"
+labels:
+  logging-infra: test
+  provider: openshift
+  component: test
+objects:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    labels:
+      run: elasticsearch-operator-e2e-test
+    name: elasticsearch-operator-e2e-test
+    namespace: ${TEST_NAMESPACE}
+  spec:
+    containers:
+    - env:
+      - name: TEST_NAMESPACE
+        value: ${TEST_NAMESPACE}
+      - name: IMAGE_ELASTICSEARCH_OPERATOR
+        value: ${IMAGE_ELASTICSEARCH_OPERATOR}
+      image: ${IMAGE_E2E_TEST}
+      imagePullPolicy: Always
+      name: elasticsearch-operator-e2e-test
+      securityContext:
+        privileged: true
```

Comment:
does this need to run as privileged?  the logging test pod does because it needs access to the node journal, /var/log files, etc.

Reply:
no, i wasn't sure if it was making a difference with why it wasn't able to be created... i'll end up removing this.